src/components/manage/Widgets/CheckboxWidget.jsx

```diff
@@ -131,13 +131,7 @@ const CheckboxWidget = ({
                 checked={value}
                 disabled={onEdit !== null}
                 onChange={(event, { checked }) => onChange(id, checked)}
-              />
-              <Label
-                basic
-                horizontal
-                as="label"
-                htmlFor={`field-${id}`}
```

Comment:
We need the `for` because of accessibility :(
Unfortunately Semantic React does that wrong :( :(
Maybe copying the `onClick` prop in the original label is better?

Reply:
True. It is mentioned in https://github.com/Semantic-Org/Semantic-UI-React/issues/3745. 
I missed that.

Reply:
Done